api/endpoints/room.md

```diff
@@ -50,6 +50,25 @@ To update a Room send JSON with updated value for one or more of the attributes.
                 "response": "Successfully removed"
             }
         }
+        
+### Create a Room [POST]
+The following attributes are required to create a Room: `name`, `floor`, and `type`.
+
++ Request (application/json)
+
+			{
+				"name": "room 101",
+				"floor": {
+					"id": 16
+				},
+				"type": {
+					"id": 44
+				}
+			}
+
++ Response 201
+
+	[Room][]
```

Comment:
If we move the Create Room block down to line 98, we can organize the sections to match similar endpoints. For example, calls to the `/rooms` endpoint belong to the Room Collection, while the `/rooms/types` endpoint are specific to room types, so can belong to their own section.

![image](https://user-images.githubusercontent.com/15133199/40129545-281f40b8-58fa-11e8-9c14-88f35788a62b.png)

Reply:
I didn't even notice the pattern among other endpoints. If not, I would have updated them all. I think having all CRUD operations per resource grouped together makes more sense. 

For example: /rooms. You see the room resource and see retrieve (Read), edit (Update) and remove (Delete). At first glance, create looks to be missing but it's found under "Room Collection" which looks out of place to me. Same goes for the rest of the create endpoints in the documentation. (Also, I'll make sure to update the URI for the create endpoints since it's incorrect due to the way the endpoints are grouped).

What do you think?

Reply:
@arturo3 Yea, good thinking--I'm curious why this organization was chosen over CRUD. If we decide on a reorganization, it will be a lower priority ticket for later I think; we're still missing documentation for entire end points like Mail, so maybe after those get added we can revisit this idea 💡

Reply:
Sounds good. I'll leave it on the back-burner for now.

Reply:
@arturo3 Okay dokey -- as soon as this block gets down to line 98, I'll go ahead and merge it 👍